app/controllers/twilio_webhooks_controller.rb

```diff
@@ -15,13 +15,20 @@ def update_outbound_call
     update_params[:call_duration] = params["CallDuration"] if params["CallDuration"].present?
     call.update(update_params)
   end
-
+  
   def dial
     @outbound_call = OutboundCall.find(params[:id])
     twiml = Twilio::TwiML::VoiceResponse.new
     twiml.say(message: 'Please wait while we connect your call.')
-    twiml.dial(number: @outbound_call.to_phone_number)
-
+    # The status callback for the call is attached to the dial event to the client.
+    # This means that the length of the call will be based on how long the user was connected to the client,
+    # And the status will be based on whether the client picked up the call.
+    twiml.dial do |dial|
+      dial.number(@outbound_call.to_phone_number,
+                  status_callback_event: 'answered completed',
```

Comment:
what does answer completed mean? is the status event always the same when we get into this block? should we save things like this status callback event as constants instead of just having the string here?

i realize a lot of these questions could probably be answered by the API documentation so maybe they are unnecessary

Reply:
I don't think they need to be constants because what we want to track could change per call we make depending on what/why we're calling and what we want to track about it.

These are a list of statuses that we're telling Twilio we want to be notified about. By default, the value for this is just "completed", but I added answered here as well so that if a call is in progress to the client, we have a note on the notes page that indicates that a call was initialized and it is in progress.